aviary/subsystems/propulsion/utils.py

```diff
@@ -252,3 +252,21 @@ def setup(self):
 #     DELTA_MAX_NOZZLE_AREA = auto()
 #     MAX_NOZZLE_AREA = auto()
 #     REF_NOZZLE_EXIT_AREA = auto()
+
+
+class PropModelVariables(Enum):
+    '''
```

Comment:
PEP 257 specifies the preference for using double quotes (""") for class/method/function docstrings.

https://peps.python.org/pep-0257/

Reply:
Changed.